symengine/lib/symengine_wrapper.pyx

```diff
@@ -2080,6 +2054,94 @@ class DenseMatrixIter(object):
 
 Matrix = DenseMatrix
 
+cdef class MutableDenseMatrix(DenseMatrix):
```

Comment:
Change `DenseMatrix` to `DenseMatrixBase` and then add an alias from `MutableDenseMatrix` to `DenseMatrix`